lib/gears/table.lua

```diff
@@ -10,12 +10,12 @@ local rtable = table
 local gmath = require("gears.math")
 local gtable = {}
 
---- Join all tables given as parameters.
--- This will iterate all tables and insert all their keys into a new table.
+--- Join all tables given as arguments.
+-- This will iterate over all tables and insert their entries into a new table.
```

Comment:
shouldn't it be some word between `all` and `tables`?

Reply:
IMHO it is fine, but maybe "all of the tables"?
Maybe the original "iterate all tables" was fine already though?!

Reply:
judging from the article which i referenced above it could be `... all the tables ...`

Reply:
@Veratil 
Thanks.  This is my impression also.
Are you a native speaker?